src/register/index.html

```diff
@@ -1,247 +1,268 @@
-            Salasanat eivät täsmää
+            <span localization-key="passwordsDontMatchError" />
           </label>
         </fieldset>
         <fieldset id="email-field">
-          <label for="email">Sähköpostiosoite (vapaaehtoinen)</label>
-          <input
-            id="email"
+          <label for="email"
+            localization-key="emailLabel"></label>
+          <input id="email"
             name="email"
             type="email"
             maxlength="200"
-            onkeyup="checkInput('email')"
-          />
+            onkeyup="checkInput('email')" />
           <label class="error-message">
             <img src="/static/icons/warning.svg" />
-            Sähköpostiosoite on virheellinen
+            <span localization-key="emailError">
+            </span>
           </label>
-          <p class="field-info">
-            Sähköpostiosoitteen tallentaminen mahdollistaa salasanan
-            palauttamisen myöhemmin. Sähköpostiosoitetta ei käytetä muihin
-            tarkoituksiin.
+          <p class="field-info"
+            localization-key="emailFieldInfo">
           </p>
         </fieldset>
         <fieldset id="display-name-field">
           <label for="display-name">
-            Julkinen nimimerkki *
+            <span localization-key="displayNameLabel"></span>
             <span class="align-right">
               <span class="tooltip">
                 <img src="/static/icons/tooltip.svg" />
-                <span class="tooltip-text">
-                  Nimimerkin tulee olla vähintään 2 merkkiä pitkä.
+                <span class="tooltip-text"
+                  localization-key="displayNameTooltip">
                 </span>
               </span>
             </span>
           </label>
-          <input
-            id="display-name"
+          <input id="display-name"
             name="display-name"
             type="text"
             maxlength="30"
             minlength="2"
             pattern=".{2,30}"
             onkeyup="checkInput('display-name')"
-            required
-          />
+            required />
           <label class="error-message">
             <img src="/static/icons/warning.svg" />
-            Nimimerkki on liian lyhyt
+            <span localization-key="displayNameTooShortError" />
           </label>
-          <p class="field-info">
-            Syötä nimimerkki, josta sinua ei voi tunnistaa, jos haluat käyttää
-            palvelua anonyymisti.
+          <p class="field-info"
+            localization-key="displayNameFieldInfo">
           </p>
         </fieldset>
-        <h2>Yksityisyydensuoja</h2>
+        <h2 localization-key="privacyTitle"></h2>
         <section>
-          <p>
-            Käytämme tietojasi ainoastaan tämän palvelun mahdollis- tamiseksi.
-            Noudatamme tarkkoja tietoturvastandardeja ja teemme parhaamme, jotta
-            tietosi säilyvät turvassa ja yksityisinä.
+          <p localization-key="privacyParagraph1">
```

Comment:
This did not seem to work in the registration-page, or it was in finnish

Reply:
The code works but I just didn't bother to translate the privacy text, because I assume it'll be replaced with the official one soon. 😄

Reply:
'We only use your data and message history to make this service possible. We follow high security standards and do our best to keep your conversations private and safe.', - found from https://github.com/sos-lapsikyla/ylitse-app/blob/master/src/localization/en.ts#L159

Reply:
Ylitse-app has a lot that could possibly be reused!

Reply:
Wow thanks!

Reply:
Privacy text translations added!